assets/src/edit-story/index.js

```diff
@@ -21,6 +21,7 @@ import Modal from 'react-modal';
 import { StrictMode } from 'react';
 import { render } from 'react-dom';
 import { FlagsProvider } from 'flagged';
+import 'web-animations-js/web-animations-next-lite.min.js';
```

Comment:
Aha! Good to know this is needed.

Should it perhaps be imported in a single location, e.g. `assets/src/animation/index.js`?

Reply:
Also, in addition to that: might be worth looking into importing all the animation stuff (or at least the polyfill) lazily. 14kB bundle size increase just for older browsers.

Reply:
yea both good calls. I'll update this to to have a single location and then create that separate ticket to make this lazily imported

Reply:
ticket moving some of animation stuff to be lazily loaded:
https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/5807